k8s_handle/k8s/availability_checker/resource_getters.py

```diff
@@ -0,0 +1,69 @@
+from typing import List, Set
+
+from k8s_handle.k8s.api_extensions import ResourcesAPI, CoreResourcesAPI
+
+VERSION_DELIMITER = "/"
+
+
+class AbstractResourceGetter(object):
+    def is_processable_version(self, api_group: str) -> bool:
+        raise NotImplementedError
+
+    def get_resources_by_version(self, api_group: str) -> Set[str]:
+        raise NotImplementedError
+
+
+class CoreResourceGetter(AbstractResourceGetter):
+
+    def __init__(self, resource_api: CoreResourcesAPI):
+        self.api = resource_api
+        self.versions = {}
+
+    def is_processable_version(self, api_group: str) -> bool:
+        return len(api_group.split(VERSION_DELIMITER, 1)) == 1
```

Comment:
https://docs.python.org/3/library/stdtypes.html#str.count
This may simplify the check.

Reply:
Or, maybe we can do split and check that split parts is not empty, to not to go to API if something like "/" or empty string is given. And add that cases in unit test.